tokens/blau.json

```diff
@@ -410,6 +410,36 @@
       "type": "color",
       "description": "blauPurple"
     },
+    "successHighInverse": {
+      "value": "{palette.blauGreen70}",
+      "type": "color",
+      "description": "blauGreen70"
+    },
+    "warningHighInverse": {
+      "value": "{palette.blauYellow70}",
+      "type": "color",
+      "description": "blauYellow70"
+    },
+    "errorHighInverse": {
+      "value": "{palette.blauRed70}",
+      "type": "color",
+      "description": "blauRed70"
+    },
+    "promoHighInverse": {
+      "value": "{palette.blauPurple}",
+      "type": "color",
+      "description": "blauPurple"
+    },
+    "brandHighInverse": {
+      "value": "{palette.blauBlueSecondary}",
+      "type": "color",
+      "description": "blauBlueSecondary"
+    },
+    "neutralMediumInverse": {
```

Comment:
could you move `neutralMediumInverse` close to neutrals?

Reply:
Done! and I have added them for all brands